src/Clinics/Domain/Actions/ListClinicAction.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Clinics\Domain\Actions;
+
+use Illuminate\Pagination\LengthAwarePaginator;
+use Lightit\Clinics\Domain\Models\Clinic;
+use Spatie\QueryBuilder\QueryBuilder;
+
+class ListClinicAction
+{
+    /**
+     * @return LengthAwarePaginator<int, Clinic>
+     */
+    public function execute(): LengthAwarePaginator
+    {
+        return QueryBuilder::for(Clinic::class)
+            ->orderBy('id', 'asc')
```

Comment:
```suggestion
            ->oldest('id')
```
This translates to `->orderBy('id', 'asc')`. You can also use `latest('id')`, which would translate to `->orderBy('id', 'desc')`.